tests/test_suppliers.py

```diff
@@ -110,3 +110,12 @@ def test_add_a_supplier(self):
     # def test_find_supplier(self):
         """ Find a supplier by ID """
         # pass
+    
+    def test_all(self):
+        Supplier(supplierName="Walmart", address="NYC", averageRating=5, productIdList = [1,2,3]).save()
+        Supplier(supplierName="Costco", address="SF", averageRating=2, productIdList = [1,3,4]).save()
+        suppliers = Supplier.all()
+        self.assertEqual(len(suppliers), 2)
+        self.assertEqual(suppliers[0].supplierName, 'Walmart')
+        self.assertEqual(suppliers[1].supplierName, 'Costco')
+        """ Return a list of suppliers """
```

Comment:
This comment explains the purpose of this method and will be shown as the document during testing and future usage. Usually, it is supposed to be placed right after the method definition.